.github/workflows/labeler.yml

```diff
@@ -0,0 +1,26 @@
+# This workflow will triage pull requests and apply a label based on the
+# paths that are modified in the pull request.
+#
+# To use this workflow, you will need to set up a .github/labeler.yml
+# file with configuration.  For more information, see:
+# https://github.com/actions/labeler
+
+name: Labeler
+on:
+  pull_request:
```

Comment:
@l0uden - is this configuration fine to use? See our discussion on Slack :)

Reply:
Yes!)